packages/vscode-extension/package.nls.json

```diff
@@ -250,6 +250,9 @@
   "teamstoolkit.handlers.provisionDescription": "[%s] is successfully created at [local address](%s). Continue to provision and then you can preview the app.",
   "teamstoolkit.handlers.provisionDescription.fallback": "[%s] is successfully created at %s. Continue to provision and then you can preview the app.",
   "teamstoolkit.handlers.provisionTitle": "Provision",
+  "teamstoolkit.handlers.manualStepRequired": "[%s] is successfully created at [local address](%s). Before start debugging, please follow the manual steps outlined in the README file.",
```

Comment:
Proposed: [%s] is created at [local address](%s). Before debugging, follow the steps in the README file.

Reply:
Updated, thanks!